__tests__/ship.test.js

```diff
@@ -0,0 +1,114 @@
+const Ship = require('../src/ship');
+const Port = require('../src/port');
+
+describe('Ship instantiation', () => {
+    let port1;
+    let itinerary;
+    let ship;
+
+    beforeEach(() => {
+        port1 = new Port('Port 1');
+        itinerary = {
+            ports: [port1],
+        };
+        ship = new Ship(itinerary);
+    });
+
+    describe('ship constructor', () => {
```

Comment:
Id say you could remove this describe block and just have the tests nested inside the describe on line 4, just seems unnecessary to have them nested twice